### ISSUE TITLE
NVDA does not read the article element in Chromium-based browsers (Edge, Google Chrome)

### ISSUE THREAD

**Issue by Marc2p** [nvaccess/nvda#15687]
### Steps to reproduce:
1) Use Edge or Chrome browser
2) Open a URL containing articles (article tag). For example, [this](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_article)
3) Navigate through the example text, starting with the level one heading that says "The article element".

### Actual behavior:
NVDA does not read the article tag, even though we have it configured in the document formatting options
### Expected behavior:
NVDA should read the article tag if we have it configured in the document formatting options
### NVDA logs, crash dumps and other attachments:
None
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2023.2 (2023.2.0.29051)
#### Windows version:
Windows 11 22H2 (build 22621.2428)
#### Name and version of other software in use when reproducing the issue:
Microsoft Edge 118.0.2088.69 (Build oficial) (64 bits) and Google Chrome 118.0.5993.118 (Build oficial) (64 bits)
#### Other information about your system:
In Mozilla Firefox it works
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA 2023.3 beta 4, same issue
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
I am wondering if the Chromium browsers expose this atribute correctly? If yes, I think it is an implementation issue in NVDA.

**Comment by seanbudd** [comment]
Closing as can't fix. This is an issue with Chromium and needs to be reported to Chrome/Google. The generated IAccessible object is not exposing the XML role for article.

**Comment by PatrykMis** [comment]
Hi, I think this issue deserves to be reopened.

I just tested this again in Chromium 141 (Brave browser), and the accessibility tree clearly exposes `<article>` with `Role: article`. So the original reason for closing (Chromium not exposing the role) is no longer valid, I think.

This means NVDA could (and should) map `article` the same way as other landmarks, instead of ignoring it. Right now, JAWS handles articles correctly, but NVDA still doesn’t, even though the role is present in the tree.

Could this be reconsidered and reopened?

**Comment by SaschaCowley** [comment]
Re-opening, as I can see the `<article>` tag when inspecting with NVDA. This should have been triaged as blocked/needs-external-fix to begin with.

**Comment by extra808** [comment]
`article` is a [document structure role](https://w3c.github.io/aria/#document_structure_roles), not a landmark role.
Nevertheless, even when 'Articles' is checked in NVDA's Document Formatting preferences (it's unchecked by default), Chrome does not mention articles, even when they're named. Using Chrome's DevTools, the Accessibility pane does show the `<article>` element's role and name (if set). Tested with NVDA 2026.1 and Chrome 148.
